package-lock.json

```diff
@@ -0,0 +1,15952 @@
+{
```

Comment:
🙈

Reply:
huh that is odd lol where did that come from lol

Reply:
oh haha this is more that just that curly lol needs to be in .gitignore huh